build.sh

```diff
@@ -10,6 +10,21 @@ yum -y install rpm-ostree selinux-policy-targeted rpm-build \
     rsync pygobject3-base python3-gobject-base \
     createrepo_c dnf-utils
 
+# Gather RPMs before we ran builddep
+rpm -qa --queryformat='%{NAME}\n' | sort -u > /root/rpms.txt
+dnf builddep -y rpm-ostree
+git clone https://github.com/jlebon/rpm-ostree
+cd rpm-ostree
+# Note --enable-rust
+./autogen.sh --prefix=/usr --libdir=/usr/lib64 --sysconfdir=/etc --enable-rust
+make -j 8
+make install
+cd ..
+rm rpm-ostree -rf
+rpm -qa --queryformat='%{NAME}\n' |sort -u > /root/rpms-new.txt
+# Yeah this is a pretty awesome hack; now we remove the BuildRequires
+(diff -u /root/rpms{,-new}.txt ||true) |grep -Ee '^\+[A-Za-z0-9]'|sed -e s,'^+,,' | xargs yum -y remove
```

Comment:
Hah, cool trick.
Although, `comm` might be better suited here... something like:

```
comm -1 -3 /root/rpms{,-new}.txt | xargs -r yum -y remove
```